app/src/main/java/org/fossasia/susi/ai/rest/services/SusiService.kt

```diff
@@ -205,6 +206,18 @@ interface SusiService {
     @GET("/cms/getSkillFeedback.json")
     fun fetchFeedback(@QueryMap query: Map<String, String>): Call<GetSkillFeedbackResponse>
 
+    /**
+     * Delete Feedback posted by the user
+     * @param query A query map consisting of following key value pairs
+     * model       Model of the skill (e.g. general)
+     * group       Group of skill (e.g. Knowledge)
+     * language    Language directory in which the skill resides (e.g. en)
+     * skill       Skill Tag of object in which the skill data resides
+     * access_token Access token of the user
+     */
+    @GET("/cms/removeFeedback.json")
```

Comment:
Why GET? As you have written `Delete Feedback posted by the user` then it should be POST

Reply:
The api accept get method. Moreover it was mentioned in gitter channel to make get request. Also get method is also present during addition of feedback. So I used it.

Reply:
@liveHarshit Yeah, SUSI server is weird that way. Instead of /api/v1/whatever_stuff, it is /aaa/whateverStuff.json

No idea why it's `aaa` or .json after every URL. 

Or why is data even sent as GET and not POST. 

Or why is access token passed as a GET parameter and not in header. Freakishly insecure way of doing things